k8s/upgrades/0.9.0-0.1.0/claim_blockdevices.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+############################################################################
+# STEP: Get all block devices present on the cluster and corresponding disk#
+#                                                                          #
+#                                                                          #
+# NOTES: Obtain the pool deployments to perform upgrade operatio           #
+############################################################################
+
+function get_csp_list() {
+    local csp_list=""
+    local spc_name=$1
+    csp_list=$(kubectl get csp -l openebs.io/storage-pool-claim=$spc_name \
+             -o jsonpath="{range .items[*]}{@.metadata.name}:{end}")
+    rc=$?
+    if [ $rc -ne 0 ]; then
+        echo "Failed to get csp related to spc $spc"
+        exit 1
+    fi
+    echo $csp_list
+}
+
+function create_bdc_claim_bd() {
+    local spc_name=$1
+    local disk_name=$2
+    local bd_name=$(echo $disk_name | sed 's|disk|blockdevice|')
```

Comment:
is this ok to rename this to `db_name_replace_fn`
I hope I got the understanding right.
This will replace disk or blockdevice occurrence with the actual disk name.
In addition, this replacement will be done later/lazy execution.

Reply:
functionality of this function is
1) Get block device name from disk name
     Ex: If disk name is `disk-1234543210` then block device name will be `blockdevice-1234543210`
2) Get details from disk CR like host name and uid
3) Fill the bdc-create.tpl.json with required values.
4) apply the populated bdc.yaml